Listener leak error with 232 version of maven-test-framework

A listener leak error has popped up when using the latest 232 version of the `maven-test-framework` dependency.  It appears the `maven-test-framework` has hardened their checks for leaks.

```
Listeners leaked for interface com.intellij.openapi.editor.ex.FocusChangeListener:
[com.intellij.codeInsight.daemon.impl.EditorTracker$1@11e9d6d5]
java.lang.AssertionError: Listeners leaked for interface com.intellij.openapi.editor.ex.FocusChangeListener:
[com.intellij.codeInsight.daemon.impl.EditorTracker$1@11e9d6d5]
	at org.junit.Assert.fail(Assert.java:89)
	at com.intellij.testFramework.EditorListenerTracker.checkListenersLeak(EditorListenerTracker.java:56)
	at com.intellij.testFramework.fixtures.impl.HeavyIdeaTestFixtureImpl.lambda$tearDown$7(HeavyIdeaTestFixtureImpl.java:158)
	at com.intellij.testFramework.RunAll$Companion$actionSequence$1$1.invoke(RunAll.kt:72)
	at com.intellij.testFramework.RunAll$Companion$actionSequence$1$1.invoke(RunAll.kt:72)
	at com.intellij.testFramework.common.RunAllKt.runAllCatching(runAll.kt:56)
	at com.intellij.testFramework.common.RunAllKt.runAll(runAll.kt:32)
	at com.intellij.testFramework.RunAll.run(RunAll.kt:21)
	at com.intellij.testFramework.fixtures.impl.HeavyIdeaTestFixtureImpl.tearDown(HeavyIdeaTestFixtureImpl.java:175)
	at com.intellij.testFramework.fixtures.impl.CodeInsightTestFixtureImpl.lambda$tearDown$43(CodeInsightTestFixtureImpl.java:1422)
	at com.intellij.testFramework.EdtTestUtil.runInEdtAndWait(EdtTestUtil.java:33)
	at com.intellij.testFramework.fixtures.impl.CodeInsightTestFixtureImpl.lambda$tearDown$44(CodeInsightTestFixtureImpl.java:1422)
	at com.intellij.testFramework.RunAll$Companion$actionSequence$1$1.invoke(RunAll.kt:72)
	at com.intellij.testFramework.RunAll$Companion$actionSequence$1$1.invoke(RunAll.kt:72)
	at com.intellij.testFramework.common.RunAllKt.runAllCatching(runAll.kt:56)
	at com.intellij.testFramework.common.RunAllKt.runAll(runAll.kt:32)
	at com.intellij.testFramework.RunAll$Companion.runAll(RunAll.kt:45)
	at com.intellij.testFramework.RunAll.runAll(RunAll.kt)
	at com.intellij.testFramework.fixtures.impl.CodeInsightTestFixtureImpl.tearDown(CodeInsightTestFixtureImpl.java:1379)
	at com.intellij.maven.testFramework.MavenTestCase.tearDownFixtures(MavenTestCase.java:296)
	at com.intellij.maven.testFramework.MavenTestCase.lambda$tearDown$15(MavenTestCase.java:203)
	at com.intellij.testFramework.EdtTestUtil.runInEdtAndWait(EdtTestUtil.java:33)
	at com.intellij.maven.testFramework.MavenTestCase.lambda$tearDown$16(MavenTestCase.java:203)
	at com.intellij.testFramework.RunAll$Companion$actionSequence$1$1.invoke(RunAll.kt:72)
	at com.intellij.testFramework.RunAll$Companion$actionSequence$1$1.invoke(RunAll.kt:72)
	at com.intellij.testFramework.common.RunAllKt.runAllCatching(runAll.kt:56)
	at com.intellij.testFramework.common.RunAllKt.runAll(runAll.kt:32)
	at com.intellij.testFramework.RunAll.run(RunAll.kt:21)
	at com.intellij.maven.testFramework.MavenTestCase.tearDown(MavenTestCase.java:211)
	at com.intellij.maven.testFramework.MavenImportingTestCase.lambda$tearDown$7(MavenImportingTestCase.java:106)
	at com.intellij.testFramework.RunAll$Companion$actionSequence$1$1.invoke(RunAll.kt:72)
	at com.intellij.testFramework.RunAll$Companion$actionSequence$1$1.invoke(RunAll.kt:72)
	at com.intellij.testFramework.common.RunAllKt.runAllCatching(runAll.kt:56)
	at com.intellij.testFramework.common.RunAllKt.runAll(runAll.kt:32)
	at com.intellij.testFramework.RunAll$Companion.runAll(RunAll.kt:45)
	at com.intellij.testFramework.RunAll.runAll(RunAll.kt)
	at com.intellij.maven.testFramework.MavenImportingTestCase.tearDown(MavenImportingTestCase.java:97)
	at com.intellij.testFramework.UsefulTestCase.invokeTearDown(UsefulTestCase.java:433)
	at com.intellij.testFramework.UsefulTestCase.defaultRunBare(UsefulTestCase.java:421)
	at com.intellij.testFramework.UsefulTestCase.lambda$runBare$12(UsefulTestCase.java:487)
	at com.intellij.testFramework.EdtTestUtil.lambda$runInEdtAndWait$1(EdtTestUtil.java:40)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:308)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:792)
	at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:739)
	at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:733)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:761)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:322)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
```

--- a/build.gradle
+++ b/build.gradle
@@ -157,11 +157,6 @@ test {
         showStandardStreams = true
         exceptionFormat = 'full'
     }
-
-    filter {
-        // disable lsp4jakarta integration tests until listener leak issue is resolved: https://github.com/OpenLiberty/liberty-tools-intellij/issues/541
-        excludeTestsMatching "io.openliberty.tools.intellij.lsp4jakarta.it.*"
-    }
 }
 
 tasks {

--- a/src/test/java/io/openliberty/tools/intellij/lsp4jakarta/it/core/BaseJakartaTest.java
+++ b/src/test/java/io/openliberty/tools/intellij/lsp4jakarta/it/core/BaseJakartaTest.java
@@ -48,9 +48,7 @@ public abstract class BaseJakartaTest extends MavenImportingTestCase {
 
     @Override
     protected void setUpFixtures() throws Exception {
-        super.setUpFixtures();
-        //myTestFixture = IdeaTestFixtureFactory.getFixtureFactory().createFixtureBuilder(getName()).getFixture();
-        //myTestFixture.setUp();
+        //Don't call super.setUpFixtures() here, that will create FocusListener leak.
         myProjectBuilder = IdeaTestFixtureFactory.getFixtureFactory().createFixtureBuilder(getName());
         final JavaTestFixtureFactory factory = JavaTestFixtureFactory.getFixtureFactory();
         ModuleFixtureBuilder moduleBuilder = myProjectBuilder.addModule(JavaModuleFixtureBuilder.class);

